docker/README.md

```diff
@@ -16,6 +16,16 @@ Follow the official instructions [here](https://docs.docker.com/compose/install/
 git clone https://github.com/google/timesketch.git
 cd timesketch
 ```
+### Verify the Kernel Settings
+Follow the official instructions [here](https://www.elastic.co/guide/en/elasticsearch/reference/6.4/docker.html#docker-cli-run-prod-mode)
+
+### Create elasticsearch directory
+
+```
+mkdir /var/lib/elasticsearch
+chown 1000:1000 /var/lib/elasticsearch
```

Comment:
1000? I guess this needs to be the docker user?

Reply:
uid,gid=1000 is the elasticsearch user inside the elasticsearch docker.  I wasn't sure how to cleanly do this without using the raw uid/gid.  I haven't tried, but I guess you could also run something like this after the docker-compose up command:

`sudo docker exec -ti docker_elasticsearch_1 chown elasticsearch:elasticsearch /var/share/elasticsearch/data`

This is the effect that is needed, but I need to see if this works before changing up the readme to this command.

Reply:
So because the docker-compose yaml file has a "restart: always" and the elasticsearch docker keeps failing because it can't create a node directory in the volume directory /var/lib/elasticsearch, there doesn't seem to be time enough to execute the command cleanly within docker before it restarts:

```
$ sudo docker exec -it docker_elasticsearch_1 chown elasticsearch:elasticsearch /usr/share/elasticsearch
Error response from daemon: Container 5efcf17ad8c6b8e78c105eb73d6f01668cbbfe8bcbf1d8d779812ef39cbe2ab0 is restarting, wait until the container is running
```

Precreating the /var/lib/elasticsearch directory and setting the owner to uid,gid=1000 seems to be the only option I can come up with.  Does someone else have a cleaner way of doing this in docker?

```
$ sudo docker exec -it docker_elasticsearch_1 ls -al /usr/share/elasticsearch/data 
total 12
drwxr-xr-x  3 elasticsearch elasticsearch 4096 Oct  7 22:34 .
drwxrwxr-x 11 elasticsearch root          4096 Oct  7 22:34 ..
drwxrwxr-x  3 elasticsearch root          4096 Oct  7 22:34 nodes

$ ls -al /var/lib/elasticsearch 
total 12
drwxr-xr-x  3 1000  1000  4096 Oct  8 09:34 .
drwxr-xr-x 83 root  root  4096 Oct  8 09:27 ..
drwxrwxr-x  3 1000  root  4096 Oct  8 09:34 nodes
```